doc/nrf/libraries/bluetooth_services/mesh/light_ctrl_srv.rst

```diff
@@ -7,6 +7,12 @@ Light Lightness Control Server
    :local:
    :depth: 2
 
+.. note::
+   This model interacts with the new sensor API introduced as of |NCS| v2.6.0.
```

Comment:
Parts of the model API have only been changed for the client, and not for the server?

Reply:
Whoops, forgot to committ the file. Will fix, two sec.